index.html

```diff
@@ -6,6 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>I-Health</title>
     <link rel="stylesheet" href="styles/home.css">
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
 </head>
 <body>
     <header>
```

Comment:
- [ ] [Optional] It would be nice to make your design identical to that of the design template. Currently, you don't have the top menu navbar. I would advise you to include this feature. It would be nice to include other links that are present in the bottom navbar even if they are placeholders. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/181916569-90e5b39f-4892-42dc-ad20-f1d46fc74c3b.png)

### Design Template

![image](https://user-images.githubusercontent.com/77106962/181916595-56557726-b6b8-45e5-8ddd-c266790aa923.png)